docs/src/examples/graf_files.md

```diff
@@ -108,15 +219,81 @@ PSRI.close(ior)
 rm(FILE_PATH; force = true)
 ```
 
-To choose the agents order use `use_header` and `header`
+## Using Graf files in a study
+
+As presented earlier, an attribute for a collection can have its data stored in a Graf file, all that being specified in the `GrafScenarios` entry of the study JSON. 
+
+If you have a Graf file that should be linked to a study, you can use the function [`PSRI.link_series_to_file`](@ref) to do so.
 
 ```@example rw_file
-data_from_file = PSRI.file_to_array(
-        PSRI.OpenBinary.Reader, 
-        FILE_PATH;
-        use_header=true,
-        header=["Agent 5", "Agent 2", "Agent 3", "Agent 4", "Agent 1"]
+PSRI.link_series_to_file(
+        data, 
+        "PSRDemandSegment", 
+        "HourDemand", 
+        "DataHourDemand",
+        PATH_TO_GRAF_FILE
     )
-@assert all(isapprox.(data_from_file[1, :, :, :], time_series_data[end, :, :, :], atol=1E-7))
-@assert all(isapprox.(data_from_file[end, :, :, :], time_series_data[1, :, :, :], atol=1E-7))
+```
+
+### GrafTable
+
+We can retrieve the data stored in a Graf file using the [`PSRI.get_graf_series`](@ref) function. This function returns a `GrafTable` object.
+
+```@example rw_file
+graf_table = PSRI.get_series(
+        data,
+        "PSRDemandSegment",
+        "HourDemand";
+        use_header = false
+    )
+```
+
+Once you have a GrafTable object, you can display it as a table in your terminal
+
+```@example rw_file
+using DataFrames
+
+DataFrame(graf_table)
+```
```

Comment:
doesnt it display without the data frames?

might be better to say display it in the terminal, show the display and then say: "we can also convert and display as a dataframe"

Reply:
Actually not. It gives a raw representation of the Table-like item.